src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_blob_scenarios.py

```diff
@@ -64,6 +64,13 @@ def verify_blob_upload_and_download(self, group, account, file_size_kb, blob_typ
         blob_name = self.create_random_name(prefix='blob', length=24)
         account_info = self.get_account_info(group, account)
 
+        # create file for uploading without --name
+        local_file_without_name = self.create_temp_file(file_size_kb)
+        blob_name_for_substitution = self.create_random_name(prefix='blob', length=24)
+        old_file_name = os.path.basename(local_file_without_name)
+        new_file_name_with_path = local_file_without_name.replace(old_file_name, blob_name_for_substitution)
+        os.rename(local_file_without_name, new_file_name_with_path)
```

Comment:
Don't forget to delete the temp file manually since after renaming, it can't be deleted automatically by test framework.

Reply:
I have changed the name back to the original on line 87.